setup.py

```diff
@@ -278,6 +282,7 @@ def check_file_at_path(path2file):
                 depends=['numba/np/ufunc/workqueue.h'],
                 include_dirs=[os.path.join(tbb_root, 'include')],
                 extra_compile_args=cpp11flags,
+                extra_link_args=extra_link_args,
```

Comment:
Why is this needed for TBB pool but not OpenMP one? Am guessing the `pthread_atfork` call in TBB and no `pthread.h` API use in OpenMP (even though the header file is `#include`d https://github.com/numba/numba/blob/9a7820cb5f21776587d23e7facf3b95fcbe6097d/numba/np/ufunc/omppool.cpp#L21 probably from adding debug code during development, e.g. calls to `pthread_self()`).

Reply:
Yes, the OpenMP extension does not appear to need the linker flags.

Reply:
Thanks for checking. I suggest removing the unused header as well as the use of the link time flags.

Reply:
I have removed the header in 5646137a2d8630d92af8426c931025dec47ad804